.github/workflows/generate-version-translations.yml

```diff
@@ -0,0 +1,128 @@
+name: Generate Version Translations
+
+on:
+    push:
+        branches: [main]
+        paths:
+            - "releases/*/versions/*.json"
```

Comment:
I believe this should target a push but only on `release/ios-*-android-*`, since we have branch protection enabled and also Github can't overcome that.
I think Github has a protection to not loop, but we'd need to verify once this PR lands on main